src/PSPackageProject.psm1

```diff
@@ -93,6 +93,102 @@ function Invoke-PSPackageProjectTest
     )
 
     ## TODO implement calling tests
+    Invoke-BinSkim
```

Comment:
This should be called when `$Type -eq 'StaticAnalysis'`

Reply:
```suggestion
    if ($Type -eq 'StaticAnalysis')
    {
        Invoke-BinSkim
    }
```